boards/msba2/Makefile.include

```diff
@@ -1,2 +1,5 @@
+# when using miniterm set RTS and DTR lines to 0, otherwise the board is reset
+MINITERMFLAGS = --rts 0 --dtr 0
```

Comment:
Better move this to `boards/common/msba2/Makefile.include` - that's where `PYTERMFLAGS` are defined too.

Reply:
You can squash directly.

Reply:
Moved to the common Makefile